smoke_tests/README.md

```diff
@@ -0,0 +1,37 @@
+# smoke_tests
+
+This directory contains [artillery.io](https://artillery.io/) configuration to run post service deployment smoke tests.
+
+These tests should to run after a deployment to ensure that things are behaving as expected in the deployed environment.
```

Comment:
```suggestion
These tests should to run after a deployment to ensure that things are behaving as expected in the deployed environment.
```

"should to run"?

Reply:
Going to fix this in an upcoming PR - fixing the docs elsewhere too.